src/function-configuration.ts

```diff
@@ -94,6 +94,8 @@ export const DEFAULT_FAILURE_POLICY: FailurePolicy = {
   retry: {},
 };
 
+export const MAX_NUMBER_USER_LABELS = 58;
```

Comment:
Does this actually need to be exported here (and thus in index.ts)? We could just put this const in cloudfunctions.ts

Reply:
We don't technically need to. However, we export other similar constants here, like MAX_TIMEOUT_SECONDS, so I figured it was appropriate to do so for this one too